### ISSUE TITLE
Sticky header does not appear after returning to list screen

### ISSUE THREAD

**Issue by VitaliBov** [davideas/FlexibleAdapter#698]
Hello. Thank you for the great library. 
I had a problem with sticky headers. I need to show a sticky header when displaying the fifth item in the list. Header disappears when you click "OK" or "Cancel". The appearance of the header and its disappearance animated. I decided to initially call the `setDisplayHeadersAtStartUp(true)` method. Header container initially invisible.
My procedure is:
1. In the onViewCreated method, the adapter and RecyclerView are initialized
2. I open the detailed screen
3. After returning to the list screen, the method onViewCreated called again, but the sticky header does not appear.

Fragment:
![1](https://user-images.githubusercontent.com/23474382/50074398-d7721b80-01ec-11e9-8cee-0f5a65be3240.jpg)
![4](https://user-images.githubusercontent.com/23474382/50075391-e6a69880-01ef-11e9-99bd-3523b1579775.jpg)

Adapter:
![5](https://user-images.githubusercontent.com/23474382/50075497-2c636100-01f0-11e9-9515-58fe94af09c3.jpg)

Item:
![3](https://user-images.githubusercontent.com/23474382/50075383-dabad680-01ef-11e9-89f6-47a996027fbe.jpg)

Header:
![2](https://user-images.githubusercontent.com/23474382/50075359-c5de4300-01ef-11e9-8f15-1f54d63cfb2b.jpg)
![7](https://user-images.githubusercontent.com/23474382/50075635-86642680-01f0-11e9-9ef2-5a100116bad6.jpg)

![ezgif com-video-to-gif 2](https://user-images.githubusercontent.com/23474382/50076008-86185b00-01f1-11e9-91b3-39648b39aac1.gif)

**Comment by davideas** [comment]
@VitaliBov, do you rebuild the fragment after returning from details? Or you push a new Activity on top?

What logs say? `Adapter.enableLogs(level..)`

**Comment by VitaliBov** [comment]
I rebuild the fragment.

At the beginning:
![image](https://user-images.githubusercontent.com/23474382/50078375-db576b00-01f7-11e9-9625-3bc5adc00161.png)
![image](https://user-images.githubusercontent.com/23474382/50078399-f0cc9500-01f7-11e9-8205-c5cef841951b.png)
![image](https://user-images.githubusercontent.com/23474382/50078444-0c37a000-01f8-11e9-9907-6685d617a923.png)
![image](https://user-images.githubusercontent.com/23474382/50078464-1bb6e900-01f8-11e9-9566-ef4da3bf6a63.png)
![image](https://user-images.githubusercontent.com/23474382/50078477-283b4180-01f8-11e9-91d4-31e02ed7d31c.png)

After returning to the list:
![image](https://user-images.githubusercontent.com/23474382/50078494-35583080-01f8-11e9-9c52-9c0150f1a664.png)
![image](https://user-images.githubusercontent.com/23474382/50078514-4143f280-01f8-11e9-8b54-d60618cf47eb.png)
![image](https://user-images.githubusercontent.com/23474382/50078557-591b7680-01f8-11e9-907c-21fef893c62d.png)

**Comment by davideas** [comment]
Thank, I will analyse the logs, but I strongly suggest to open a new Activity, at least it resolves immediately your issue.

**Comment by VitaliBov** [comment]
@davideas Yes, when using Activity, the problem does not occur. We use architecture with Single Activity, and we would like to implement everything in fragments.
